projects/storefrontapp/server.ts

```diff
@@ -33,6 +35,7 @@ export function app(): express.Express {
   const indexHtml = existsSync(join(distFolder, 'index.original.html'))
     ? join(distFolder, 'index.original.html')
     : join(distFolder, 'index.html');
+  const indexHtmlContent = readFileSync(indexHtml, 'utf-8');
```

Comment:
I suggest to do this job in our uber-error-handler, so customers would give us onlu the indexHtml path:

```
server.use(defaultServerErrorResponseHandlers({indexHtml}))
```
(perhaps the long name is yet up to discussion)

Note: I suggest named params inside an object to allow for easisly adding new params in the future potentially